common/djangoapps/entitlements/api/v1/filters.py

```diff
@@ -34,8 +34,8 @@ def __init__(self, name='uuid', label=None, widget=None, method=None, lookup_exp
 class CourseEntitlementFilter(filters.FilterSet):
 
     uuid = UUIDListFilter()
-    course_uuid = UUIDListFilter()
```

Comment:
Do we not also want to be able to filter by the course the entitlement is for?

Reply:
We should add as needed in my opinion.

Reply:
Also @McKenzieW This was dead code that should have been removed in the original PR. It is a little more complicated to add this and I think it will require a custom FilterSet.